_includes/tutorials/produce-consume-lang/scala/markup/dev/2_0-create-build-file.adoc

```diff
@@ -0,0 +1,7 @@
+Create the following Sbt build file, named `build.sbt` at the root of the project folder:
```

Comment:
"at the root of the project folder" might be confusing for the student as there is a `project` subfolder.  At this stage the users terminal is at `produce-consume-scala`, so I suggest we just give them the file name:

```suggestion
Create the following Sbt build file, named `build.sbt` for the project:
```